src/dotnet/APIView/APIViewWeb/Repositories/CommentsManager.cs

```diff
@@ -19,14 +25,48 @@ public class CommentsManager
 
         private readonly NotificationManager _notificationManager;
 
+        private readonly OrganizationOptions _Options;
+
+        public HashSet<GithubUser> TaggableUsers;
+
         public CommentsManager(
             IAuthorizationService authorizationService,
             CosmosCommentsRepository commentsRepository,
-            NotificationManager notificationManager)
+            NotificationManager notificationManager,
+            IOptions<OrganizationOptions> options)
         {
             _authorizationService = authorizationService;
             _commentsRepository = commentsRepository;
             _notificationManager = notificationManager;
+            _Options = options.Value;
+
+            TaggableUsers = new HashSet<GithubUser>();
+
+            LoadTaggableUsers();
```

Comment:
This is fine for now. Only catch is that it is populated only during startup of server. APIView uses singleton class that's created at startup. This is not a blocker for now and we can improvise it later.

Reply:
you mean improve? :)